source/PlayerInfo.cpp

```diff
@@ -687,6 +687,19 @@ int64_t PlayerInfo::Salaries() const
 
 
 
+
+// Calculate the daily maintenance cost for all unparked ships.
+int64_t PlayerInfo::Maintenance() const
+{
+	int64_t maintenance = 0;
+	for(const shared_ptr<Ship> &ship : ships)
+		if(!ship->IsParked() && !ship->IsDestroyed())
+			maintenance += ship->Attributes().Get("maintenance cost");
```

Comment:
Does this work properly if I have 3 outfits with maintenance costs on a ship (does it automatically add all up)?

Reply:
`Ship::Attributes` returns all the attributes on a ship, including those from outfits, as `Ship::AddOutfit` adds the attributes of the outfits to the attributes of the ship.

Reply:
Okay, then this works as intended.